{"filename": "haskell-enclosed-exceptions-1.0.3-302-x86_64.pkg.tar.zst", "name": "haskell-enclosed-exceptions", "base": "haskell-enclosed-exceptions", "version": "1.0.3-302", "desc": "Catching all exceptions from within an enclosed computation", "csize": "13530", "isize": "40189", "md5sum": "5141d5a8ce5b9569b8857a8ccff3ce94", "sha256sum": "a6f8f5267c57c3ee0da9fdb27bfcb6a8cd651ce40bafeb3b03a676297e0b64b1", "pgpsig": "iQIyBAABCgAdFiEEtZcfLFwQqaCMYAMPeGxj8zDXy5IFAmisn+EACgkQeGxj8zDXy5JY4g/4zvURjs53wEL1DQ/nWVMN4GqSOVu3sgDi9mR/oUS0A0+QtWCmGt2266wpw9C1vNpWwCjqQfVQQoVtJV0cchTssVpSE0e8S/7KocuTcAJORTFwsmZgXFIkvaTFSsQddQLkVlh8ZSkpvCxCNfAX1kGFAqJwjlcX6/gfNIr4PNYcuJTt5J4slgEFUnNnNa0yseNGNgKOpkpnlfwHx+i9LdlvxBLcvngO6aswKo8IO83xAJI3kUzEfycvOgwYLRmGxdByU240FqR5R394SltFl0TV+m2Sf4a5ebyIDikgbumHDbG7l9nfAyWfxZhhadli+qKsH2rvVRYF69kRTdEtD/AGFSrZLOZsujRiyrDWrZ33vvEXuYFjikhSJwkzm1ymPJiCu6XxpA/HseOilaB0WGdNWpWFuMbN3Ky5crwLOZbBywhVQPnMwYLTzhnHkcrXx9iE7xeWE15M0fZQsxnvTnRQ2S6eD/OyiMudyfc1JxfCfYKfQky4AiSB2Fv5hBt2qo/yE9z9xYOwAJmEu94DiNSkk69SMwv0tyKPGdMq9lhcoDCVkMJc1bka+bi4JEY09/6rf/E4TNfbZHoczNpK1nqpQiBWt6nuaH7ELi4PchT7NDsJIKCg9AqLDrQnX1/8YWGwNCJBTCXiU3DlfpdXU7tfunRQ2Mzd/NyrKMRyIIkwOQ==", "url": "https://github.com/jcristovao/enclosed-exceptions", "license": "MIT", "arch": "x86_64", "builddate": "1756143567", "packager": "Felix Yan <felixonmars@archlinux.org>", "depends": ["ghc-libs", "haskell-lifted-base", "haskell-monad-control", "haskell-transformers-base"], "makedepends": ["ghc"]}